.evergreen/config.yml

```diff
@@ -1180,8 +1223,24 @@ tasks:
       - func: "fetch crypt_shared"
       - func: "bootstrap mongo-orchestration"
       - func: "run kmip server"
+      - func: "build csfle expansions"
       - func: "run csfle tests"
 
+  - name: "test-csfle-serverless"
+    tags: ["csfle", "serverless"]
+    commands:
+      - command: expansions.update
```

Comment:
I have _no dang idea_ why evergreen doesn't have expansions set when it runs this task.  If I had to guess, it would be that there's some internal state reset between `task_group` tasks that clears them, so they persist from setup into the `test-serverless` task but not in this one.

Reply:
weird. I don't understand evergreen expansions very well at all so I'm not sure I can help answer this either 🙂